### PR TITLE
add free/member filter to worldhopper plugin

### PR DESCRIPTION
**Pull request by DrRobosnarfen**
SubFilterDemo

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java [hunk 1]
```diff
@@ -149,6 +149,11 @@
 	private Instant lastFetch;
 	private boolean firstRun;
 
+	public static final int BOTH = 2;
+	public static final int MEMBERS = 1;
+	public static final int FREE = 0;
+	protected int filterMode;
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java]: This should probably just be a `SubscriptionFilterMode` in the panel itself.

**Comment by Adam-** [comment]
There is also #8021 which is relevant to you maybe - but it is highly over-engineered imo.

**Comment by DrRobosnarfen** [comment]
Moved the filtermode check to the panel. And yeah, I want to keep it simple.

**Comment by Adam-** [comment]
Added in 1a73893065ea7eae835c4dfb58275812f9a5258e

**Pull request closed without merging**